### PR TITLE
HOWTO-make-a-release.md: take into account the moved newsflash.txt

### PR DESCRIPTION
**Pull request by levitte**
~Related to openssl/otc#1~

### CODE REVIEW

**Comment by levitte** [comment]
Only the last commit applies here.  The other commit comes from #119

**Comment by levitte** [comment]
This is pending an actual move

File: HOWTO-make-a-release.md [hunk 1]
```diff
@@ -229,24 +245,36 @@ with $TOOLS, and is generally called like this:
 
 The manual for that script is found in `$TOOLS/release-tools/MKRELEASE.md`
 
+## Update the newsflash locally
+
+*The changes in this section should be made in your clone of the openssl
+otc repo*
```
Comment (t8m) [inline HOWTO-make-a-release.md:251]: omc/data

File: HOWTO-make-a-release.md [hunk 2]
```diff
@@ -229,24 +245,36 @@ with $TOOLS, and is generally called like this:
...
+
+Update the release-data/newsflash.txt file.  This normally is one or two
+lines.  Just copy and paste existing announcements making minor changes for
+the date and version number as necessary.  If there is an advisory then
+ensure you include a link to it.
+
+*Do* send the commits to the reviewer and await their approval.
+
+Commit your changes, but *do not push* them to the otc repo at this stage.
+(the otc repo being `git@github.openssl.org:omc/data.git`)
```
Comment (t8m) [inline HOWTO-make-a-release.md:261]: omc/data instead of otc

**Comment by levitte** [comment]
vulnerabilities.xml and secadv were moved too, so we adjust that as well.
At this point, we're not touching the web repository at all any more.

**Comment by levitte** [comment]
Note that #119 is merged, I rebased this on top of fresh master

**Pull request closed without merging**